datalad_registry/blueprints/api/url_metadata.py

```diff
@@ -1,13 +1,15 @@
-# This file is for defining the API endpoints related to URL metadata,
+# This file is for defining the API endpoints related to dataset URL metadata,
 # i.e. the metadata of datasets at individual URLs.
 
 from datalad_registry.com_models import URLMetadataModel
 from datalad_registry.models import URLMetadata, db
 
 from . import bp
 
+_URL_PREFIX = "/url-metadata"
 
-@bp.get("/url-metadata/<int:url_metadata_id>")
+
+@bp.get(f"{_URL_PREFIX}/<int:url_metadata_id>")
```

Comment:
not sure what the point of this "no functionality change" diff, with also such a loud PR title, but ok -- let's proceed ;)